testing/CMakeLists.txt

```diff
@@ -55,6 +55,9 @@ if (CDAT_BUILD_MATPLOTLIB)
   add_subdirectory(matplotlib)
 endif()
 
+# Test ipython
+add_test(run_ipython ${CMAKE_INSTALL_PREFIX}/bin/ipython -V)
```

Comment:
@aashish24 this is what I was referring to - not certain it works as I'd guessed